geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloFeatureWriter.scala

```diff
@@ -142,8 +117,7 @@ abstract class AccumuloFeatureWriter(sft: SimpleFeatureType,
       feature
     }
 
-    val toWrite = new FeatureToWrite(withFid, defaultVisibility, encoder, indexValueEncoder)
-    writers.foreach(write => write(toWrite))
+    writer(new FeatureToWrite(withFid, defaultVisibility, encoder, indexValueEncoder))
```

Comment:
Does introducing FeatureToWrite objects cost us anything in terms of object instantiation/GC?